packages/server/package.json

```diff
@@ -15,14 +15,14 @@
     "express": "4.18.1",
     "jsdom": "20.0.0",
     "jsdom-global": "3.0.2",
-    "pdfmake": "0.2.5",
+    "pdfmake": "^0.2.6",
```

Comment:
we should always use the direct version to avoid surprising updates during clean `yarn install?